docs/product-mapping.yaml

```diff
@@ -0,0 +1,22 @@
+## File used in modular documentation generator to 
```

Comment:
I don't like this being in the root of the docs directory as it's really a tooling thing, it's not related to the content.